.github/workflows/docker.yml

```diff
@@ -31,9 +31,14 @@ jobs:
           compose-file: ./docker-compose.yaml
           services: base
           up-flags: --build
+          cwd: ${{ github.workspace }}
       # - name: Verifying the container was built
       #   run: docker exec roscon24-workshop-base-1 colcon test
       # TODO: We only really want to run test on our problems. I need to write some ...
+      - name: Test pyrobosim
+        run: |
+          docker exec roscon24-workshop-base-1 bash -c "source /opt/ros/jazzy/setup.bash && colcon build"
```

Comment:
Check whether this `colcon build` is needed, as the entrypoint should already build when the container is started

Reply:
You are right, it is not needed